### PR TITLE
oscap docker http header fix

### PR DESCRIPTION
**Pull request by ybznek**
Fix case sensitivity issue while getting HTTP header dict.
+ refactoring
+ add "warning" before message

https://github.com/OpenSCAP/openscap/issues/304

I will test it later.

[SKIP_CI]

### CODE REVIEW

**Comment by ybznek** [comment]
Test passed.

File: utils/oscap_docker_python/get_cve_input.py [hunk 1]
```diff
@@ -84,11 +84,23 @@ def _fetch_single(self, dist):
             seconds_epoch = (remote_dt - epoch).total_seconds()
             utime(dest_file, (seconds_epoch, seconds_epoch))
         except KeyError:
-            stderr.write("Response header of HTTP doesn't contain" \
-                  "\"last-modified\" field. Cannot determine version" \
-                  " of remote file \"{0}\"".format(dist_url))
+            self._print_no_last_modified_warning(dist_url)
```
Comment (mpreisler) [inline utils/oscap_docker_python/get_cve_input.py]: Don't we also want to check for `self.DEBUG` here? Or even check for it inside the method.

Comment (ybznek) [inline utils/oscap_docker_python/get_cve_input.py]: Yes, you are right. I will check it inside the method.

File: utils/oscap_docker_python/get_cve_input.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ class getInputCVE(object):
     def __init__(self, fs_dest, DEBUG=False):
         ''' Simple init declaration '''
         self.dest = fs_dest
-        self.DEBUG = DEBUG
+        self.DEBUG = True
```
Comment (jan-cerny) [inline utils/oscap_docker_python/get_cve_input.py]: Is this intentional or just a oversight? I think if we set this to true, we don't have to check for it (`if self.DEBUG:`) further in the code. I can't find any other place in the code where this variable is set.

Comment (mpreisler) [inline utils/oscap_docker_python/get_cve_input.py]: Definitely an oversight. Good catch.

Comment (ybznek) [inline utils/oscap_docker_python/get_cve_input.py]: Eh, sorry for that. I have to more check more my pull requests.

**Pull request merged**